app/assets/stylesheets/barchart.scss

```diff
@@ -0,0 +1,46 @@
+$gap: 5px;
+$totalRows: 101;
+
+.barchart {
+  width: 90%;
+  height: 250px;
```

Comment:
Shouldn't this be some `em` and/or %?  or if we need to force a value does `min-height` work (with `height`)?